### DISCUSSION TITLE
Is there a way to set active organization in `authMiddleware` or other backend code?

### DISCUSSION THREAD

**Discussion by wuweiweiwu** [clerk/javascript#2587] [Support request - DO NOT OPEN HERE]
We have a Next.js app router app and are using the clerk sdk. Based on user's subscription tier we create an organization for them programmatically and on subsequent logins they're switched to the organization on client app load using the `useOrganizationList` hook

Is there a way to set the active organization in the backend using the Node.js SDK?

Would love if we could somehow do this in the `authMiddleware` `afterAuth` callback

**Comment by jescalan** [comment]
At the moment there isn't, but you can drop an upvote for the feature [in our roadmap](https://feedback.clerk.com/roadmap/39f0e922-2986-4c13-ae3f-6dc3faa5f24b)!
